bcdi/preprocessing/bcdi_utils.py

```diff
@@ -1219,7 +1219,7 @@ def reload_bcdi_data(
     # check and load kwargs
     valid.valid_kwargs(
         kwargs=kwargs,
-        allowed_kwargs={"logger", "photon_threshold"},
+        allowed_kwargs={"logger", "photon_threshold", "logfile"},
```

Comment:
This kwargs is not necessary, I don't understand why you want to add it. Instead, remove line 419 (`logfile=logfile`) in preprocessing.process_scan

```suggestion
        allowed_kwargs={"logger", "photon_threshold"},
```